<!DOCTYPE html>
<html lang="en">

<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="description" content="an introduction to working with methylation data and epigenome-wide associatation studies">
  <meta name="generator" content="Hugo 0.20.7" />

  <title>Workshop - Methlyation and EWAS &middot; ...Down the Rabbit Hole...</title>

  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/pure/0.6.2/pure-min.css">

  <!--[if lte IE 8]>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/pure/0.6.2/grids-responsive-old-ie-min.css">
  <![endif]-->
  <!--[if gt IE 8]><!-->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/pure/0.6.2/grids-responsive-min.css">
  <!--<![endif]-->

  <!--[if lte IE 8]>
  <link rel="stylesheet" href="../../../../css/side-menu-old-ie.css">
  <![endif]-->
  <!--[if gt IE 8]><!-->
  <link rel="stylesheet" href="../../../../css/side-menu.css">
  <!--<![endif]-->

  <link rel="stylesheet" href="../../../../css/blackburn.css">

  
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="../../../../css/academicons.css"/>

  
  <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet" type="text/css">

  
  

  

  <link rel="shortcut icon" href="../../../../img/favicon.ico" type="image/x-icon" />

  
  

</head>


<body>
<div id="layout">

  
<a href="#menu" id="menuLink" class="menu-link">
  
  <span></span>
</a>
<div id="menu">

  

  <div class="pure-menu">
    <ul class="pure-menu-list">
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="../../../../about/"><i class='fa fa-user fa-fw'></i>About</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="../../../../"><i class='fa fa-newspaper-o fa-fw'></i>Blog</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="../../../../projects/"><i class='fa fa-cogs fa-fw'></i>Projects</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="../../../../workshops-tutorials/"><i class='fa fa-question-circle fa-fw'></i>Tutorials</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="../../../../publications/"><i class='fa fa-list fa-fw'></i>Publications</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="../../../../presentations/"><i class='fa fa-picture-o fa-fw'></i>Presentations</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="https://sirselim.github.io/Miles_CV/"><i class='fa fa-file-pdf-o fa-fw'></i>CV</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="https://github.com/sirselim"><i class='fa fa-github fa-fw'></i>GitHub</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="https://scholar.google.co.nz/citations?user=iEhBavEAAAAJ&amp;hl=en"><i class='ai ai-google-scholar fa-fw'></i>GoogleScholar</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="https://www.researchgate.net/profile/Miles_Benton"><i class='ai ai-researchgate fa-fw'></i>ResearchGate</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="https://www.researchgate.net/profile/Miles_Benton"><i class='fa fa-linkedin-square fa-fw'></i>LinkedIn</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="https://twitter.com/miles_benton"><i class='fa fa-twitter-square fa-fw'></i>Twitter</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="http://orcid.org/0000-0003-3442-965X"><i class='ai ai-orcid fa-fw'></i>ORCiD</a>
      
        </li>
      
      
        <li class="pure-menu-item">
          <a class="pure-menu-link" href="https://www.mendeley.com/profiles/miles-benton/"><i class='ai ai-mendeley fa-fw'></i>Mendeley</a>
      
        </li>
      
    </ul>
  </div>

  <div class="pure-menu social">
  <ul class="pure-menu-list">

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

  </ul>
</div>


  <div>
  <div class="small-print">
    <small>&copy; 2017. All rights reserved.</small>
  </div>
  <div class="small-print">
    <small>Built with&nbsp;<a href="https://gohugo.io/" target="_blank">Hugo</a> & <a href="http://rmarkdown.rstudio.com/">RMarkdown</a></small>
    <small>Theme&nbsp;<a href="https://github.com/yoshiharuyamashita/blackburn" target="_blank">Blackburn</a></small>
  </div>
</div>

</div>


  <div id="main">


<div class="header">
  <h1>Workshop - Methlyation and EWAS</h1>
  <h2>an introduction to working with methylation data and epigenome-wide associatation studies</h2>
</div>
<div class="content">

  <div class="post-meta">

  <div>
    <i class="fa fa-calendar fa-fw"></i>
    <time>31 May 2017, 10:00</time>
  </div>

  

  
  
  
  

  
  
  
  

</div>

  

<p>This is a basic EWAS tutorial, introducing concepts for analysing methylation data.</p>

<p>There is a webpage version of the tutorial located <a href="https://sirselim.github.io/methylation_EWAS_workshop/">here</a>. You can load this and follow along as you run through this on your computer, if you wish to copy paste code please feel free to do so.</p>

<h2 id="data">Data</h2>

<p>The workshop uses public Illumina 450K data from several studies (GSE40279, GSE48472, GSE41114, GSE42700, GSE46573, GSE48472, GSE50586). It is currently designed to investigate the methylation differences between blood (whole blood) and buccal (cheek swabs) tissues. It is aimed at giving a very basic overview of the structure and type of data used in EWAS as well as a few of the available tools and methods to explore this data.</p>

<p>##Required software and packages</p>

<p>You will need to ensure you have the following installed on your system:</p>

<ul>
<li>the latest version of <code>R</code>: <a href="https://cran.r-project.org/">https://cran.r-project.org/</a></li>
<li><code>RStudio</code> (optional but recommended): <a href="https://www.rstudio.com/products/rstudio/download2/">https://www.rstudio.com/products/rstudio/download2/</a></li>
<li>the <code>R</code> package minfi: <a href="https://www.bioconductor.org/packages/release/bioc/html/minfi.html">https://www.bioconductor.org/packages/release/bioc/html/minfi.html</a>

<ul>
<li>to install <code>minfi</code> run the following within <code>R</code>/<code>RStudio</code>:

<ul>
<li><code>source(&quot;https://bioconductor.org/biocLite.R&quot;)</code></li>
<li><code>biocLite(&quot;minfi&quot;)</code></li>
</ul></li>
<li>the <code>R</code> package FactoMineR

<ul>
<li><code>install.packages('FactoMineR')</code></li>
</ul></li>
</ul></li>
</ul>

<p><strong>NOTE:</strong> this workshop should work across all OS (Windows, Linux, Mac) as long as the above are correctly installed.</p>

<h2 id="set-up">Set up</h2>

<h3 id="1-obtain-the-workshop-files">1. obtain the workshop files</h3>

<p>Once the above software and <code>R</code> packages are installed set up is as simple as downloading or cloning this repository, it contains all the data and scripts required for the workshop.</p>

<h4 id="download">Download</h4>

<p>You can download this repository as a zip file (see <a href="https://github.com/sirselim/methylation_EWAS_workshop">https://github.com/sirselim/methylation_EWAS_workshop</a>) and extract to the folder <code>methylation_EWAS_workshop</code> where appropriate on your system.</p>

<h4 id="clone">Clone</h4>

<p>If you are familiar with git and GitHub you can clone into this repository:</p>

<p><code>git clone https://github.com/sirselim/methylation_EWAS_workshop.git</code></p>

<h3 id="2-move-to-the-correct-directory-and-start-working">2. move to the correct directory and start working</h3>

<p>Open <code>R</code>/<code>RStudio</code> and ensure that your working directory is set to the methylation_EWAS_workshop directory, then open the <code>EWAS_analysis.R</code> script and follow along.</p>


  
<div class="prev-next-post pure-g">
  <div class="pure-u-1-24" style="text-align: left;">
    
    <a href="../../../../2015/06/26/gwas-workshop/"><i class="fa fa-chevron-left"></i></a>
    
  </div>
  <div class="pure-u-10-24">
    
    <nav class="prev">
      <a href="../../../../2015/06/26/gwas-workshop/">Introduction to Genomewide Association Studies</a>
    </nav>
    
  </div>
  <div class="pure-u-2-24">
    &nbsp;
  </div>
  <div class="pure-u-10-24">
    
  </div>
  <div class="pure-u-1-24" style="text-align: right;">
    
  </div>
</div>



  

</div>

</div>
</div>
<script src="../../../../js/ui.js"></script>




</body>
</html>

