<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>.huge[Clinical Genomics]</title>
    <meta charset="utf-8" />
    <link href="libs/tile-view/tile-view.css" rel="stylesheet" />
    <script src="libs/tile-view/tile-view.js"></script>
    <link href="libs/animate.css/animate.xaringan.css" rel="stylesheet" />
    <link href="libs/tachyons/tachyons.min.css" rel="stylesheet" />
    <link href="libs/panelset/panelset.css" rel="stylesheet" />
    <script src="libs/panelset/panelset.js"></script>
    <link rel="stylesheet" href="libs/remark-css/example_mod.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# .huge[Clinical Genomics]
## Customisable Analysis Workflows for Identification of Clinically Relevant Genetic Variants - An Update
### <div class="line-block"><a href="http://sirselim.github.io/">Dr Miles Benton</a><br />
Senior Scientist Bioinformatics<br />
Human Genomics, Institute of Environmental Science and Research (ESR)<br />
<br />
.large[Otago Genetics Seminar, 24<small><sup>th</sup></small> August 2020]<br />
<br />
<br />
.center[<img src="images/ESR_SFC_logo.png" style="width: 25%; border: 3px solid white;"/>]</div>

---

class: middle hide_logo



<style type="text/css">
.xaringan-extra-logo {
  width: 138px;
  height: 87px;
  z-index: 0;
  background-image: url(https://upload.wikimedia.org/wikipedia/commons/6/66/ESR_SFC_logo.png);
  background-size: contain;
  background-repeat: no-repeat;
  position: absolute;
  right:1.5em;bottom:1px;
}
</style>
<script>(function () {
  let tries = 0
  function addLogo () {
    if (typeof slideshow === 'undefined') {
      tries += 1
      if (tries < 10) {
        setTimeout(addLogo, 100)
      }
    } else {
      document.querySelectorAll('.remark-slide-content:not(.title-slide):not(.inverse):not(.hide_logo)')
        .forEach(el => { el.innerHTML += '<a href="https://www.esr.cri.nz/" class="xaringan-extra-logo"></a>' })
    }
  }
  document.addEventListener('DOMContentLoaded', addLogo)
})()</script>

# .center[Advocate for reproducible research...]

## .center[Where possible my presentations and code are available online]

&lt;br /&gt;

&lt;p&gt;
.center[
&lt;img src="images/github_logo.png" style="width: 520px; margin-right: 1%; margin-top: 1.5em;"/&gt;
&lt;img src="images/sirselim_qrcode.png" style="width: 202px; margin-right: 1%; margin-top: 1.5em;"/&gt;
]
&lt;/p&gt;

&lt;br /&gt;

.center[[sirselim.github.io/presentations](http://sirselim.github.io/presentations)]

&lt;!-- SVG Icons are from https://simpleicons.org --&gt;
&lt;footer class="pv3 ph3 ph5-ns tc"&gt;
  &lt;a class="link near-black hover-blue dib h2 w2 mr3" href="https://github.com/sirselim" title="GitHub"&gt;
    &lt;svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="1.414"&gt;&lt;path d="M8 0C3.58 0 0 3.582 0 8c0 3.535 2.292 6.533 5.47 7.59.4.075.547-.172.547-.385 0-.19-.007-.693-.01-1.36-2.226.483-2.695-1.073-2.695-1.073-.364-.924-.89-1.17-.89-1.17-.725-.496.056-.486.056-.486.803.056 1.225.824 1.225.824.714 1.223 1.873.87 2.33.665.072-.517.278-.87.507-1.07-1.777-.2-3.644-.888-3.644-3.953 0-.873.31-1.587.823-2.147-.083-.202-.358-1.015.077-2.117 0 0 .672-.215 2.2.82.638-.178 1.323-.266 2.003-.27.68.004 1.364.092 2.003.27 1.527-1.035 2.198-.82 2.198-.82.437 1.102.163 1.915.08 2.117.513.56.823 1.274.823 2.147 0 3.073-1.87 3.75-3.653 3.947.287.246.543.735.543 1.48 0 1.07-.01 1.933-.01 2.195 0 .215.144.463.55.385C13.71 14.53 16 11.534 16 8c0-4.418-3.582-8-8-8"/&gt;&lt;/svg&gt;
  &lt;/a&gt;
  &lt;a class="link hover-blue near-black dib h2 w2 mr3" href="https://twitter.com/miles_benton" title="Twitter"&gt;
    &lt;svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="1.414"&gt;&lt;path d="M16 3.038c-.59.26-1.22.437-1.885.517.677-.407 1.198-1.05 1.443-1.816-.634.375-1.337.648-2.085.795-.598-.638-1.45-1.036-2.396-1.036-1.812 0-3.282 1.468-3.282 3.28 0 .258.03.51.085.75C5.152 5.39 2.733 4.084 1.114 2.1.83 2.583.67 3.147.67 3.75c0 1.14.58 2.143 1.46 2.732-.538-.017-1.045-.165-1.487-.41v.04c0 1.59 1.13 2.918 2.633 3.22-.276.074-.566.114-.865.114-.21 0-.416-.02-.617-.058.418 1.304 1.63 2.253 3.067 2.28-1.124.88-2.54 1.404-4.077 1.404-.265 0-.526-.015-.783-.045 1.453.93 3.178 1.474 5.032 1.474 6.038 0 9.34-5 9.34-9.338 0-.143-.004-.284-.01-.425.64-.463 1.198-1.04 1.638-1.7z" fill-rule="nonzero"/&gt;&lt;/svg&gt;
  &lt;/a&gt;
  &lt;a class="link hover-blue near-black dib h2 w2 mr3" href="https://scholar.google.co.nz/citations?user=iEhBavEAAAAJ&amp;hl=en" title="Google Scholar"&gt;
    &lt;svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="1.414"&gt;&lt;path d="M12 24a7 7 0 1 1 0-14 7 7 0 0 1 0 14zm0-24L0 9.5l4.838 3.94A8 8 0 0 1 12 9a8 8 0 0 1 7.162 4.44L24 9.5z" fill-rule="nonzero"/&gt;&lt;/svg&gt;
  &lt;/a&gt;
  &lt;a class="link hover-blue near-black dib h2 w2 mr3" href="http://orcid.org/0000-0003-3442-965X" title="ORCID"&gt;
    &lt;svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="1.414"&gt;&lt;path d="M12 0C5.372 0 0 5.372 0 12s5.372 12 12 12 12-5.372 12-12S18.628 0 12 0zM7.369 4.378c.525 0 .947.431.947.947s-.422.947-.947.947a.95.95 0 0 1-.947-.947c0-.525.422-.947.947-.947zm-.722 3.038h1.444v10.041H6.647V7.416zm3.562 0h3.9c3.712 0 5.344 2.653 5.344 5.025 0 2.578-2.016 5.025-5.325 5.025h-3.919V7.416zm1.444 1.303v7.444h2.297c3.272 0 4.022-2.484 4.022-3.722 0-2.016-1.284-3.722-4.097-3.722h-2.222z" fill-rule="nonzero"/&gt;&lt;/svg&gt;
  &lt;/a&gt;
&lt;/footer&gt;

---
class: top

# acknowledgements

&lt;div class="dt dt--fixed"&gt;
  &lt;div class="dtc tc pv4 bg-white"&gt;
    &lt;article class="mw5 center bg-white br3 pa3 pa4-ns mv3 ba b--black-10"&gt;
      &lt;div class="tc"&gt;
        &lt;img src="images/Leah.jpeg" class="br-100 h4 w4 dib ba b--black-05 pa2" title="Photo of team member"&gt;
        &lt;h4 class="f3 mb2"&gt;Leah Kemp&lt;/h4&gt;
        &lt;h4 class="f5 fw4 gray mt0"&gt;(ESR)&lt;/h4&gt;
      &lt;/div&gt;
    &lt;/article&gt;
  &lt;/div&gt;
  &lt;div class="dtc tc pv4 bg-white"&gt;
    &lt;article class="mw5 center bg-white br3 pa3 pa4-ns mv3 ba b--black-10"&gt;
      &lt;div class="tc"&gt;
        &lt;img src="images/Joep.jpeg" class="br-100 h4 w4 dib ba b--black-05 pa2" title="Photo of team member"&gt;
        &lt;h4 class="f3 mb2"&gt;Joep de Ligt&lt;/h4&gt;
        &lt;h4 class="f5 fw4 gray mt0"&gt;(ESR)&lt;/h4&gt;
      &lt;/div&gt;
    &lt;/article&gt;
  &lt;/div&gt;
  &lt;div class="dtc tc pv4 bg-white"&gt;
    &lt;article class="mw5 center bg-white br3 pa3 pa4-ns mv3 ba b--black-10"&gt;
      &lt;div class="tc"&gt;
        &lt;img src="images/Donia.png" class="br-100 h4 w4 dib ba b--black-05 pa2" title="Photo of team member"&gt;
        &lt;h4 class="f3 mb2"&gt;Donia Macartney-Coxson&lt;/h4&gt;
        &lt;h4 class="f5 fw4 gray mt0"&gt;(ESR)&lt;/h4&gt;
      &lt;/div&gt;
    &lt;/article&gt;
  &lt;/div&gt;
&lt;/div&gt;

---
class: middle

# Talk Outline

[1]. &lt;span style="color:#3498DB"&gt;**One year ago...**&lt;/span&gt;
  * my journey → ESR → VCF-DART  

[2]. &lt;span style="color:#3498DB"&gt;**Current day**&lt;/span&gt;
  * wfl → GPUs → lots of data

[3]. &lt;span style="color:#3498DB"&gt;**Scout tool**&lt;/span&gt;
  * {Interactive demo}

[4]. &lt;span style="color:#3498DB"&gt;**Moving forwards?**&lt;/span&gt;

---
class: middle inverse

# QUT (February 2014 - June 2018)

## Brisbane (QLD), Australia

### &lt;span style="color:#3498DB"&gt;PostDoc Research Fellow&lt;/span&gt;

&lt;p&gt;
.center[
&lt;img src="images/qut_ihbi.jpg" style="width: 240px; margin-right: 1%; margin-top: 1.5em; border: 3px solid white;"/&gt;
&lt;img src="images/qut_gardens.jpg" style="width: 240px; margin-right: 1%; margin-top: 1.5em; border: 3px solid white;"/&gt;
]
&lt;/p&gt;

---
class: inverse

background-image: url("images/proton2.jpg")
background-size: cover

---
class: middle

# Diagnostics Lab

.large[Gene panel and Exome Sequencing (NATA accredited - neurological disorders)]

&lt;br&gt;

## Sanger -&gt; Gene panel -&gt; Exome

* move to WES -&gt; increase in variants
* manual approach through 30-60K rows of data in **Excel**
* increased time to identify and validate variant(s)

---
class: middle inverse

# VCF-DART - what it's not...

It is **NOT** a variant calling and quality control pipeline

... it's about data

---

# Overview

&lt;p&gt; 
.center[&lt;img src="images/Pipeline_flowchart_v2.png" style="width: 85%; margin-right: 1%; margin-top: -0.5em; border: 3px solid white;"/&gt;]
&lt;/p&gt;

---

# Variant tiers

&lt;p&gt; 
.center[&lt;img src="images/tiers.png" style="width: 55%; margin-right: 1%; margin-top: -0.5em; border: 3px solid white;"/&gt;]
&lt;/p&gt;

---
class: middle inverse

# "Module 1" - VCF-DART

.large[(&lt;span style="color:#3498DB"&gt;**V**&lt;/span&gt;ariant &lt;span style="color:#3498DB"&gt;**C**&lt;/span&gt;all &lt;span style="color:#3498DB"&gt;**F**&lt;/span&gt;ormat - &lt;span style="color:#3498DB"&gt;**D**&lt;/span&gt;iagnostic &lt;span style="color:#3498DB"&gt;**a**&lt;/span&gt;nd &lt;span style="color:#3498DB"&gt;**R**&lt;/span&gt;eporting &lt;span style="color:#3498DB"&gt;**T**&lt;/span&gt;ool)]

---
layout: false

background-image: url("images/vcfdart_1.png")
background-size: contain

---
layout: false

background-image: url("images/vcfdart_2.png")
background-size: contain

---
class: middle

# Run-time

.huge[30-50K variants (avg Proton exome VCF):]

  - 6-10 mins (24 core 256GB RAM)
  - 17-23 mins (4 core 12GB RAM)

---
class: middle inverse

# "Module 2" - VCF-DART Viewer

---
layout: false

background-image: url("images/backup_1.png")
background-size: contain

---
layout: false

background-image: url("images/backup_2.png")
background-size: contain

---
layout: false

background-image: url("images/backup_3.png")
background-size: contain

---
layout: false

background-image: url("images/backup_4.png")
background-size: contain

---
layout: false

background-image: url("images/backup_5.png")
background-size: contain

---
layout: false

background-image: url("images/backup_6.png")
background-size: contain

---
class: 

&lt;p&gt; 
.center[&lt;img src="images/nectarcloud.png" style="width: 70%; margin-right: 1%; margin-top: -0.5em; border: 3px solid white;"/&gt;]
&lt;/p&gt;

Spun up VM to give reviewers a chance to ~~break~~ test things.

* using 6 public exomes (1000G)

&lt;br&gt;

--

.large[**Allocated VM:**]

* Ubuntu 18.08
* 4 cores
* 12GB RAM
* 30GB root
* 120GB ephemeral disk

--

&lt;br&gt;

...so it can scale down to quite reasonable specs...

---
class: middle

## VCF-DART in the lab

* much quicker time to variant shortlist and curation 
* can get to validation sooner (if required)
* meaning report delivered faster

## Summary

* open source
* modular (tools and databases)
* scalable (laptops -&gt; servers)
* easy to deploy and use [soon to become easier]
* versioning 

---
class: middle inverse

## Current state of play (as of a year ago...)

* exciting times!

* cancer genomics - Auckland University (Prof Cristin Print), Genomics Aotearoa

* merging 'pipelines' to offer a flexible, unified and freely distributed system across NZ

* accreditation

&lt;p&gt;
.center[
&lt;img src="images/UoA.png" style="width: 272px; margin-right: 1%; margin-top: 1.5em; border: #FFFFFF  3.5px outset;"/&gt;
&lt;img src="images/GA-Wide-Colour-1200px.jpg" style="width: 315px; margin-right: 1%; margin-top: 1.5em; border: #FFFFFF  3.5px outset;"/&gt;
]
&lt;/p&gt;

---
class: middle

## Availability / Deployment

&lt;br&gt;

.large[Working on a wide range of deployment options:]

&lt;br&gt;

&lt;p&gt;
.center[
&lt;img src="images/github.jpg" style="width: 200px; margin-right: 2%;"/&gt;
&lt;img src="images/aws.png" style="width: 215px; margin-right: 2%;"/&gt;
&lt;img src="images/singularity-logo.svg" style="width: 135px; margin-right: 2%; margin-left: 2%;"/&gt;
&lt;img src="images/NESI-logo.jpg" style="width: 300px; margin-right: 2%;"/&gt;
]
&lt;/p&gt;

---
class: middle inverse

# to-do

* &amp;#10004; ~~submit~~ manuscript - accepted (The Journal of Molecular Diagnostics) 
* &amp;#10004; [release to the wild - GitHub (anyone interested can help out)](https://github.com/sirselim/WES_ShinyDiscover)
* finish implementing ClinGen &amp; ClinVar links
* option for GnomAD-beta
* refactoring main code base (snakemake)
* complete docker/singularity version
* continue developing documentation!
* work with clinicians to further develop and refine

---
class: middle inverse

# to-do (SPOILER: the goal posts have shifted!)

* &amp;#10004; ~~submit~~ manuscript - accepted (The Journal of Molecular Diagnostics) 
* &amp;#10004; ~~[release to the wild - GitHub (anyone interested can help out)](https://github.com/sirselim/WES_ShinyDiscover)~~
* ~~finish implementing ClinGen &amp; ClinVar links~~
* ~~option for GnomAD-beta~~
* ~~refactoring main code base (snakemake)~~
* ~~complete docker/singularity version~~
* ~~continue developing documentation!~~
* ~~work with clinicians to further develop and refine~~

---
class: middle inverse

# What a difference a year makes...

.huge[&lt;span style="color:#3498DB"&gt;**Human Genomics at ESR**&lt;/span&gt;]

---

# What do we do?

* our own research

&lt;br&gt;

* contract(s) with CCDHB Wellington Regional Genetics Lab

&lt;br&gt;

* providing support to CCDHB for Clinical Genomics GA project (Stephen Robertson)

&lt;br&gt;

* clinical research collaboration with local clinician (Richard Carroll) - hyperparathyroidism
  * sequencing 100 exomes
  * engaging clinicians from other DHBs
  * exploring the replacement of gene panels with exome/genome

---
class: top

# Complete Overhaul! [Leah Kemp]

&lt;br&gt;

&lt;article class="cf"&gt;
  &lt;div class="fl w-100 w-50-ns bg-white"&gt;
    &lt;br&gt;
    &lt;br&gt;
    &lt;br&gt;
    &lt;ul&gt;
      &lt;li&gt;Alignment and Variant Calling Workflow&lt;/li&gt;
        &lt;ul&gt;
          &lt;li&gt;&lt;a href="https://github.com/ESR-NZ/human_genomics_pipeline" target="blank"&gt;[human_genomics_pipeline]&lt;/a&gt;&lt;/li&gt;
        &lt;/ul&gt;
    &lt;br&gt;
    &lt;br&gt;
      &lt;li&gt;Variant Annotation Workflow&lt;/li&gt; 
        &lt;ul&gt;
          &lt;li&gt;&lt;a href="https://github.com/ESR-NZ/vcf_annotation_pipeline" target="blank"&gt;[annotation_pipeline]&lt;/a&gt;&lt;/li&gt;
        &lt;/ul&gt;
    &lt;/ul&gt;
  &lt;/div&gt;
  &lt;div class="fl w-100 w-50-ns bg-white tc"&gt;
  &lt;article class="mw5 center bg-white br3 pa3 pa4-ns mv3 ba b--black-10"&gt;
    &lt;div class="tc"&gt;
      &lt;img src="images/Leah.jpeg" class="br-100 h4 w4 dib ba b--black-05 pa2" title="Photo of team member"&gt;
      &lt;h4 class="f3 mb2"&gt;Leah Kemp&lt;/h4&gt;
      &lt;h4 class="f5 fw4 gray mt0"&gt;(ESR)&lt;/h4&gt;
    &lt;/div&gt;
  &lt;/article&gt;
  &lt;/div&gt;
&lt;/article&gt;

.center[.huge[ESR GitHub: [https://github.com/ESR-NZ](https://github.com/ESR-NZ)]]

---
class: top

# Current situation

* we have automated, reproducible, deployable pipelines
  - workflow language, containers and package managers

&lt;br&gt;

* depending on the resources you deploy on you can get fairly reasonable "turn-around"
  - 8-12 hours per 100X coverage exome
  - ~30-40 hours for a 30X coverage genome
    - time obviously increases with coverage

&lt;br&gt;

* having access to decent compute means these can run in parallel
  - so depending on resources you might:
      + run 10 exomes at once
      + run 4 - 8 genomes at once
  - this means you get more finishing in the time it takes to process one

---
class: middle inverse

# We can go faster!

&lt;br&gt;

.huge[... thanks to eResearch 2020]

---
class: middle

# ...enter [Nvidia Clara Parabricks](https://www.nvidia.com/en-us/docs/parabricks/)

&lt;article class="cf"&gt;
  &lt;div class="fl w-50 bg-white tc"&gt;
    &lt;img src="images/parabricks.png" style="width: 50%; margin-right: 1%; margin-top: 0em;"/&gt;
  &lt;/div&gt;
  &lt;div class="fl w-50 bg-white tc"&gt;
    &lt;img src="images/nvidia.png" style="width: 60%; margin-right: 1%; margin-top: 2.5em;"/&gt;
  &lt;/div&gt;
&lt;/article&gt;

---
class: middle

.center[
&lt;img src="images/tesla_v100.jpg" style="width: 70%; margin-right: 1%; margin-top: 0em;"/&gt;
]

---
class: middle inverse

.center[
&lt;img src="images/parabricks_pipelines.png" style="width: 70%; margin-right: 1%; margin-top: 0em;"/&gt;
]

&lt;br&gt;

.center[[www.nvidia.com/en-us/healthcare/clara-parabricks/](https://www.nvidia.com/en-us/healthcare/clara-parabricks/)]

---

## Germline Pipeline

.center[
&lt;img src="https://www.nvidia.com/content/dam/en-zz/Solutions/parabricks/Parabricks-germline-pipeline.png" style="width: 70%; margin-right: 1%; margin-top: 0em;"/&gt;
]

---

## Somatic Pipeline

.center[
&lt;img src="https://www.nvidia.com/content/dam/en-zz/Solutions/parabricks/Parabricks-somatic-pipeline.png" style="width: 50%; margin-right: 1%; margin-top: 0em;"/&gt;
]

---

# ... so what ? ...

--

.center[.large[exact same workflow, but THE SPEED!]]

--

&lt;br&gt;

&lt;article class="cf"&gt;
  &lt;div class="fl w-100 w-50-ns bg-white"&gt;
    &lt;ul&gt;
    &lt;li&gt;&lt;b&gt;Pipeline [CPU]&lt;/b&gt;&lt;/li&gt;
      &lt;ul&gt;
        &lt;li&gt;Exomes: &gt;100X exome takes between 8-12 hours&lt;/li&gt;
        &lt;li&gt;Genomes: ~30X genome takes between 30-48 hours&lt;/li&gt;
      &lt;/ul&gt;
    &lt;/ul&gt;
    &lt;br&gt;
    &lt;ul&gt;
    &lt;li&gt;&lt;b&gt;Pipeline [GPU - Parabricks]&lt;/b&gt;&lt;/li&gt;
      &lt;ul&gt;
        &lt;li&gt;Exomes: &gt;100X exome takes between &lt;span style="color:#3498DB"&gt;&lt;b&gt;6-12 mins&lt;/b&gt;&lt;/span&gt;&lt;/b&gt;&lt;/li&gt;
        &lt;li&gt;Genomes: 100X genome takes &lt;span style="color:#3498DB"&gt;&lt;b&gt;~4 hours&lt;/b&gt;&lt;/span&gt;&lt;/li&gt;
      &lt;/ul&gt;
    &lt;/ul&gt;
  &lt;/div&gt;
  &lt;div class="fl w-100 w-50-ns bg-white tc"&gt;
    &lt;img src="images/fast-turtle.png" style="width: 70%; margin-right: 1%;"/&gt;
  &lt;/div&gt;
&lt;/article&gt;

---

# ... so what ? ...

&lt;br&gt;

&lt;article class="cf"&gt;
  &lt;div class="fl w-100 w-50-ns bg-white"&gt;
    &lt;ul&gt;
    &lt;li&gt;[ESR] This is running on 2x Nvidia Tesla V100 GPUs&lt;/li&gt;
      &lt;ul&gt;
        &lt;li&gt;~$15,000 NZD per card&lt;/li&gt;
      &lt;/ul&gt;
    &lt;/ul&gt;
    &lt;br&gt;
    &lt;ul&gt;
    &lt;li&gt;Can have up to 8 cards in a node&lt;/li&gt;
      &lt;ul&gt;
        &lt;li&gt;this results in processing a genome in &lt;b&gt;~30 mins&lt;/b&gt;&lt;/li&gt;
      &lt;/ul&gt;
    &lt;/ul&gt;
  &lt;/div&gt;
  &lt;div class="fl w-100 w-50-ns bg-white tc"&gt;
    &lt;img src="images/v100rack.jpg" style="width: 70%; margin-right: 1%;"/&gt;
  &lt;/div&gt;
&lt;/article&gt;

--

&lt;br&gt;

.center[.large[*&lt;span style="color:#3498DB"&gt;&lt;b&gt;( "cough" ... someone should ask why we don't just all buy GPUs ... "cough" )&lt;/b&gt;&lt;/span&gt;*]]

---

# Revisit: How does this change what we do?

* our own research

&lt;br&gt;

* contract(s) with CCDHB Wellington Regional Genetics Lab

&lt;br&gt;
  
* providing support to CCDHB for Clinical Genomics GA project (Stephen Robertson)

&lt;br&gt;
  
* clinical research collaboration with local clinician (Richard Carroll) - hyperparathyroidism
  * sequencing 100 exomes
  * engaging clinicians from other DHBs
  * exploring the replacement of gene panels with exome/genome

---
class: middle inverse

# So what about the actual clincial side of things?!

Variant Annotation Workflow: [annotation_pipeline](https://github.com/ESR-NZ/vcf_annotation_pipeline)

---

&lt;article class="cf"&gt;
  &lt;div class="fl w-50 bg-white tc"&gt;
    &lt;img src="https://github.com/ESR-NZ/vcf_annotation_pipeline/raw/master/rulegraph.png" style="width: 60%; margin-right: 1%; margin-top: 2.5em;"/&gt;
  &lt;/div&gt;
  &lt;div class="fl w-50 bg-white"&gt;
    &lt;br&gt;
    &lt;br&gt;
    &lt;br&gt;
    &lt;ul&gt;
      &lt;li&gt;dbSNP&lt;/li&gt;
      &lt;li&gt;dbNSFP&lt;/li&gt;
      &lt;li&gt;VEP&lt;/li&gt;
      &lt;li&gt;GENMOD&lt;/li&gt;
        &lt;ul&gt;
          &lt;li&gt;inheritance models&lt;/li&gt;
          &lt;li&gt;CADD scores&lt;/li&gt;
          &lt;li&gt;&lt;b&gt;ranked score*&lt;/b&gt;&lt;/li&gt;
        &lt;/ul&gt;
    &lt;/ul&gt;
    &lt;br&gt;
    &lt;br&gt;
    &lt;i&gt;&lt;sup&gt;*&lt;/sup&gt;this is pretty nifty&lt;/i&gt;
  &lt;/div&gt;
&lt;/article&gt;

---
class: top

# Ranked Score?

* able to write custom filter config for [GENMOD](https://github.com/moonso/genmod) software
  
* filter can be a user defined combination of criteria

* our current minimal:
  - Filter = PASS
  - allele freq
  - CADD score
  - inheritance model
  
* the higher score 'floats' to the top when viewed in [Scout](https://github.com/Clinical-Genomics/scout)

* easy to extend (i.e. conservation, SIFT, PROVEAN, ...)

---
class: middle

&lt;div class="flex items-center justify-center pa4 bg-lightest-blue navy"&gt;
  &lt;svg class="w1" data-icon="info" viewBox="0 0 32 32" style="fill:currentcolor"&gt;
    &lt;title&gt;info icon&lt;/title&gt;
    &lt;path d="M16 0 A16 16 0 0 1 16 32 A16 16 0 0 1 16 0 M19 15 L13 15 L13 26 L19 26 z M16 6 A3 3 0 0 0 16 12 A3 3 0 0 0 16 6"&gt;&lt;/path&gt;
  &lt;/svg&gt;
  &lt;h2 class="lh-title ml3"&gt;What is Scout?&lt;/h2&gt;
&lt;/div&gt;

### Answer: A free, open-source tool to analyze VCFs and collaborate on solving rare diseases quicker.

Scout documentation: [link](http://www.clinicalgenomics.se/scout/)

&lt;br&gt;

&lt;ul&gt;
&lt;li&gt;&lt;b&gt;Simple&lt;/b&gt; - Analyze variants in a simple to use web interface.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;Aggregation&lt;/b&gt; - Combine results from multiple analyses and VCFs into a centralized database.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;Collaboration&lt;/b&gt; - Write comments and share cases between users and institutes.&lt;/li&gt;
&lt;/ul&gt;

---
class: middle inverse

# Time to jump into Scout...

---
class: middle inverse

# Future thinking?

* wider advertising of talks/presentations

* collaborative meet-ups

* hands-on workshops
  * explore options with GA / NeSi?

* ...

---
class: middle hide_logo

&lt;article class="center mw5 mw6-ns br3 hidden ba b--black-10 mv4"&gt;
  &lt;h1 class="f4 bg-near-white br3 br--top black-60 mv0 pv2 ph3"&gt;Thank you&lt;/h1&gt;
  &lt;div class="pa3 bt b--black-10"&gt;
    &lt;p class="f6 f5-ns lh-copy measure"&gt;
      Thank you for your time today, any questions?
    &lt;/p&gt;
  &lt;/div&gt;
&lt;/article&gt;
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
