<!doctype html>
<html lang="en">

    <!-- meta data and javascript import -->
	<head>
		<meta charset="utf-8">
		<title>Identification of allelic-specific methylation profiles in a genetic isolate</title>
		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/base.css">
		<link rel="stylesheet" href="css/theme/black.css" id="theme">
		<!-- Code syntax highlighting -->
		<link rel="stylesheet" href="lib/css/zenburn.css">
		<!-- Make sure jquery is loaded before highcharts -->
		<!-- Jquery not required for Highstock charts which has lightweight jquery embeded -->
    	<!-- <script type="text/javascript" src="js/jquery-1.11.3.min.js"></script> -->
    	<!-- <script type="text/javascript" src="js/highcharts.js"></script> -->
		<!-- Import data.js to allow importing of csv files -->
		<!-- had issues with cross origin requests when accessing data stored locally -->
		<!-- Not currently implemmented in this presentation -->
		<script type="text/javascript" src="modules/data.js"></script>
		<!-- Creates dark themed plots throughout the presentation -->
		<script src="js/themes/gray.js"></script>
	</head>

    <!-- body content -->
    <body>

		<div class="reveal">

			<header class='headings'>
				<span></span>
			</header>

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
                <!-- title slide -->
  				      <section>
					          <h3>Allelic-specific methylation profiles in a genetic isolate</h3>
					          <p style="font-size: 20px">
						        <!-- <small> -->
						        <a href="http://sirselim.github.io/" target="blank">Miles Benton</a>
                    <!-- </small> -->
                    </p>
                    <p style="font-size: 14px">
                    School of Medical Sciences, Institute of Health and Biomedical Innovation, Queensland University of Technology, Brisbane, Australia <br>
                    </p>
                    <p style="font-size: 20px">
                      <a href="http://www.abacbs.org/" target="blank">AB<sup>3</sup>ACBS 2016 - QUT, Brisbane, Australia</a>
					          </p>
                    <p style="font-size: 16px">
                      1<sup style="font-size: 10px">st</sup>-2<sup style="font-size: 10px">nd</sup> November 2016
					          </p>
                  <center>
                    <img src="images/ihbi_building.jpg" width=40% />
                  </center>
                </section>

                <section>
                	<h3>My presentation available online</h3>
                	<br />
                	<p>
  					  <img src="images/github_logo.png" width=510px>
                	  <img src="images/ASM_pres_QR_code.png" width=202px />
                	</p><br />
                	<a href="http://sirselim.github.io/presentations/AB3ACBS_2016_presentation.html" target="blank">http://sirselim.github.io/presentations.html</a>
                </section>

                <!-- Norfolk Island  -->
                <!--- Vertical slides take affect here following double section breaks -->
                <section>
                  <section>
                    <h2>Norfolk Island</h2>
                    <br />
                    ...and the Norfolk Island Health Study...<br>
                  </section>
                  <section>
                    <img src="images/NI_map.png" width=60% />
                    <p style="font-size: 20px"><a href="https://github.com/gringer/bioinfscripts/blob/master/perlshaper.pl" target="blank">https://github.com/gringer/bioinfscripts/blob/master/perlshaper.pl</a></p>
                  </section>
                  <section>
                    <img src="images/NorfolkIsland.png">
                  </section>
                  <section>
                    <img src="images/NIpedigree.png"><br />
                    ~6000 member pedigree<br />
                    <p style="font-size: 14px">Macgregor S <i>et al.,:</i> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2987173/" target="blank"><i>Legacy of mutiny on the Bounty: founder effect and admixture on Norfolk Island.</i></a> Eur J Hum Genet. 2010; 18: 67–72.</p>
                  </section>
                  <section>
                	<h3>Mitochondrial Ancestry</h3>
                    <img src="images/NI_ped_mt.png"><br />
                      40% of current population haplogroup <span style="color:lightblue">B4a1a[...]</span><br />

                      <p style="font-size: 14px">Benton MC <i>et al.,:</i> <a href="https://investigativegenetics.biomedcentral.com/articles/10.1186/s13323-015-0028-9" target="blank"><i>“Mutiny on the Bounty”: the genetic history of Norfolk Island reveals extreme gender-biased admixture.</i></a> Investigative Genetics 2015, 6:11.</p>
                  </section>
                </section>

                <!-- Allelic-specific methylation  -->
                <!--- Vertical slides take affect here following double section breaks -->
                <section>
                  <section>
                    <h2>Allele-specific methylation</h2>
                    (very recent data)<br /><br />
                    <p style="font-size: 20px">
                    Other contributors: <span style="color:lightblue">Rod Lea, Donia Macartney-Coxson, Nicole White, Daniel Kennedy, Heidi Sutherland, Larisa Haupt, Kerrie Mengersen, Lyn Griffiths</span>
                    </p>
                    <center>
                      <img id="rr" src="images/Rod.jpg">
                      <img id="db" src="images/Donia.jpg">
                      <img id="dv" src="images/Nicole.jpg">
                    </center>
                  </section>
                  <section>
                    <h2>Allele-specific methylation</h2>
                    <br />
                    <p>
                  	Allele-specific  methylation  (ASM)  occurs  when  the  same  cytosine  is  differentially  methylated  on  the  <span style="color:lightblue"><b>two  alleles</b></span>  of  a  diploid organism  
                  	</p>
                  	<br />
                  	<p>
                  	ASM is a major mechanism of <span style="color:lightblue"><b>genomic imprinting</b></span>, & aberrations can lead to disease
                  	</p>
                  </section>
                  <section>
                    <h5>Identification of allele-specific methylation profiles across generations</h5>
                    <span style="color:lightblue">Proof of principle pilot study</span><br />
                    <p>
                    <ul>
                      <li>measuring genome-wide allele-specific methylation (ASM)</li>
                        <ul>
                          <span style="font-size: 75%">
                          <li>NGS bisulphite sequencing</li> 
                          <li>SeqCap Epi CpGiant (Illumina HiSeq)</li>
                          </span>
                        </ul>
                        <br />
                      <li>collected data for <span style="color:lightblue"><b>24</b></span> NI individuals</li> 
                        <ul>
                          <span style="font-size: 75%">
                          <li>comprising a close 3 generation pedigree</li>
                          </span>
                        </ul>
                        <br />
                      <li>currently generating data for another <span style="color:lightblue"><b>~90</b></span> samples</li>
                    </ul>
                    </p>
                  </section>
                  <section>

                    <ul>
                      <li>Fully customised QC and analysis pipeline:<span style="color:lightblue"><b>*</b></span></li>
                        <ul>
                          <li>fastqc, trimgalore</li>
                          <li>bismark, sambamba, picard tools</li> 
                          <li>PileOMeth, R and methylkit</li>
                        </ul>
                        <br />
                      <li>parallel processing enabled for local and remote machines</li>
                        <br />
                      <li>filtered at minimum 10 counts</li> 
                    </ul>
                    <p>
                      <span style="color:lightblue; font-size: 75%"><b>*<i>once wrangled into shape scripts will be accessible via GitHub</i></b></span>
                    </p>
                  </section>
                  <section>
                    <h3>Initial metrics</h3>
                    <ul>
                      <li>coverage <span style="color:lightblue">~40x</span> ave</li>
                      <li>lowest number of CpGs for a given sample is <span style="color:lightblue">2.67M</span></li>
                      <li>highest number of CpGs for a given sample is <span style="color:lightblue">7.52M</span></li>
                      <li>the average across all samples is <span style="color:lightblue">3.48M</span></li>
                      <li>on-target mapping rate across the 24 samples >95% average</li><br>
                    </ul>
                    <br />
                      <span style="color:lightblue"><b>~1.12M CpG sites in common across the 24 samples with at least 10 times coverage and on-target</b></span>
                  </section>
                  <section>
                    <h4>QC</h4>
                    <center>
                      <img id="genderPCA" src="images/NI_gender_PCA.png" width=85% />
                    </center>
                  </section>
                  <section>
                    <h4>QC</h4>
                    <center>
                      <img id="sampleQC" src="images/NI6828_SNP_compare_160816.png" width=95% />
                    </center> 
                    Good concordance between Illumina arrays and BS-seq                
                  </section>
                  <section>
                    <h2>Allelic-specific methylation</h2>
                    <ul>
                    <li>for an initial look-see used methpipe suite <span style="font-size: 75%"><sup>[1]</sup></span><br></li>
                    <p>
                      <ul style="font-size: 24px;">
                      <li>estimates ASM at each CpG site based on ratios of methylation</li>
                      <li>uses CpG pairs (assumes 'linkage')</li>
                      </ul>
                    </p>
                    <li>moving forward we want to use read data:</li>
                    <p>
                      <ul style="font-size: 24px;">
                        <li>assign parent of origin and then infer ASM</li>
                        <li>gives the actual info rather than just knowing where an ASM event is</li>
                      </ul>
                    </p>
                    <li>promising method recently published for mice ASM <span style="font-size: 75%"><sup>[2]</sup></span>
                  </ul>
                  	<p style="text-align: left; line-height: 18px">
                  	<span style="font-size: 35%"><sup>[1]</sup>Fang F, <i>et al.,</i>: <a href="http://www.pnas.org/content/109/19/7332.full" target="blank"><i>Genomic landscape of human allele-specific DNA methylation.</i></a> PNAS, 2012; 109: 7332–7.</span><br />
           	        <span style="font-size: 35%"><sup>[2]</sup>Krueger F, <i>et al.,</i>: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4934512/" target="blank"><i>SNPsplit: Allele-specific splitting of alignments between genomes with known SNP genotypes.</i></a> F1000Research, 2016; 5: 1479.</span>
           	        </p>
                  </section>
                  <section>
                    <h3>ASM regions</h3>
                    <ul>
                      <li>Using a custom clustering method we identified ~1800 ASM regions (AMRs) conserved within the pedigree</li><br />
                      <li>Many of these AMRs map to known and predicted imprinted genes</li>
                    </ul>
                  </section>
                  <section>
                    <h2>Visualising ASM</h2>
                    <center>
                      <img id="chr7_ASM" src="images/example_asm_chr7.png" width=85% />
                    </center>
                      <p style="text-align: center; font-size: 18px;">
                      Documented and predicted human imprinted loci are displayed in <span style="color:darkred"><b>darkred</b></span>.
                    </p>
                  </section>
                  <section>
                    <center>
                      <img id="movie" src="images/scatter_movie.gif" width=60% />
                    </center>
                    <p style="text-align: center; font-size: 18px;">
                      ASM plot of chromosome 20 for a nuclear family (<span style="color:darkred"><b>father</b></span>, <span style="color:pink"><b>mother</b></span>, <span style="color:lightblue"><b>son</b></span>, <span style="color:green"><b>daughter</b></span>)
                    </p>
                  </section>
                  <section>
                  <h3>Presence of SNPs</h3>
                  <ul>
                  <li><span style="color:lightblue"><b>1,127,867</b></span> total methylation sites</li>                    
                  <li><span style="color:lightblue"><b>444,330</b></span> have a SNP present on C/G (<span style="color:lightblue"><b>39.4%</b></span>)</li>
                  </ul>
                  <p>
                  ...however...
                  </p>
                  <p>
                   Considering MAF:
                  <ul>
                    <li><span style="color:lightblue"><b>231,452</b></span> SNPs have recorded MAF info</li>
                    <li><span style="color:lightblue"><b>12,670</b></span> SNPs >= 0.05 MAF</li>
                    <li><span style="color:lightblue"><b>26,935</b></span> SNPs >= 0.01 MAF</li>
                  </ul> 
                  </p>
                  </section>
                  <section>
                    <center>
                      <img id="chr15_SNP" src="images/ASM_chr15_SNP_MAF.png" width=85% />
                    </center>
                  </section>
                  <section>
                    <center>
                      <img id="chr20_SNP" src="images/ASM_chr20_SNP_MAF.png" width=85% />
                    </center>
                  </section>
                  <section>
                    <center>
                      <img id="chr21_SNP" src="images/ASM_chr21_SNP_MAF.png" width=85% />
                    </center>
                  </section>
                  <section>
                    <center>
                      <img id="ploschr21" src="images/plos_chr21.png" width=85% />
                    </center>
                    <span style="font-size: 35%">Alves Da Silva AF, <i>et al.,</i>: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27100087" target="blank"><i>Trisomy 21 alters DNA methylation in parent-of-origin-dependent and -independent manners.</i></a> PLoS One, 2016; 11: e0154108.</span>
                  </section>
                  <section>
                  	<h3>What about sites which aren't conserved?</h3>
                  </section>
                  <section>
                  	<h4>Nice sanity check of method (X chromosome)</h4>
                  	<img src="images/NI_SeqCap_ASMbias_chrX.png" width=75% />
                  </section>
                  <section>
                  	<img src="images/NI_SeqCap_ASMbias_chr1.png" width=65% />
                  </section>
                  <section>
                    <h3>Epigenetic inheritance</h3>
                    Next thing is to move onto modeling the inheritance...<br />
                    ...with the help of our brilliant Bayesian statisticians!<br>
                  </section>
                </section>

                <!-- summary slide -->
                <section>
                  <h2>Summary</h2>
                  <ul>
                    <li>expanding upon our multi-layered NI data</li><br>
                    <li>custom pipeline developed to interrogate bisulfite sequence data</li><br>
                    <li>initial ASM region and imprinting loci identification</li>
                    <ul>
                    <span style="font-size: 75%">
                    	<li>potential to identify previously unreported imprinted loci</li>
                    	<li>in a position to start inheritance modeling</li>
                    </span>
                    </ul>
                  </ul>          
                </section>

                <!-- acknowledgments  -->
                <section>
                  <h3>acknowledgments</h3>
                  <style type="text/css">
                  .tg  {border-collapse:collapse;border-spacing:0;border:none;margin:0px auto;}
                  .tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 20px;border-style:solid;border-width:0px;overflow:hidden;word-break:normal;}
                  .tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 20px;border-style:solid;border-width:0px;overflow:hidden;word-break:normal;}
                  .tg .tg-e3zv{font-weight:bold}
                  </style>
                  <table class="tg" style="undefined;table-layout: fixed; width: 780px">
                  <colgroup>
                  <col style="width: 260px">
                  <col style="width: 260px">
                  <col style="width: 260px">
                  </colgroup>
                    <tr>
                      <th class="tg-e3zv">QUT</th>
                      <th class="tg-e3zv">STDOI @ UTRGV<br></th>
                      <th class="tg-e3zv">NZ collaborators<br></th>
                    </tr>
                    <tr>
                      <td class="tg-031e">Lyn Griffiths<br></td>
                      <td class="tg-031e">John Blangero<br></td>
                      <td class="tg-031e"><span style="color:lightblue"><b>Donia Macartney-Coxson (ESR)</b></span><br></td>
                    </tr>
                    <tr>
                      <td class="tg-031e"><span style="color:lightblue"><b>Rod Lea</b></span><br></td>
                      <td class="tg-031e">Joanne Curran<br></td>
                      <td class="tg-031e">David Eccles (Gringene Bioinformatics)<br></td>
                    </tr>
                    <tr>
                      <td class="tg-031e"><span style="color:lightblue"><b>Larisa Haupt</b></span><br></td>
                      <td class="tg-031e">Harald Goring<br></td>
                      <td class="tg-031e">Geoff Chambers (VUW)</td>
                    </tr>
                    <tr>
                      <td class="tg-031e"><span style="color:lightblue"><b>Heidi Sutherland</b></span><br></td>
                      <td class="tg-031e"><br></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">Michelle Hanna<br></td>
                      <td class="tg-031e"><br></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">...the rest of the GRC IHBI lab group</td>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e"><span style="color:lightblue"><b>Nicole White</b></span></td>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e"><span style="color:lightblue"><b>Daniel Kennedy</b></span></td>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">Kerrie Mengersen</td>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"></td>
                    </tr>
                  </table>
                  <br />
                  <table class="tg" style="undefined;table-layout: fixed; width: 780px">
                  <colgroup>
                  <col style="width: 260px">
                  <col style="width: 260px">
                  <col style="width: 260px">
                  </colgroup>
                    <tr>
                      <td class="tg-031e">Garvan Institute of Medical Research / Kinghorn Centre for Clinical Genomics</td>
                      <td class="tg-031e">Melanie Carless (Texas Biomedical Research Institute)</td>
                      <td class="tg-031e">NHMRC</td>
                    </tr>
                    <tr>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"><br>Claire Bellis (Genome Institute of Singapore)</td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"><br>Greg Gibson (Georgia Institute of Technology, USA)</td>
                      <td class="tg-031e"></td>
                    </tr>
                  </table><br />
                  <span style="color:lightblue; font-size:20px"><b>The people of Norfolk Island who who volunteered for this study.</b></span>
                </section>

                  <section>
                    <center>
                      <img id="labgroup" src="images/IMG_1127.JPG" width=85% />
                    </center>
                  </section>

                <!-- last slide, questions -->
                <section>
                  <h2>Thank you</h2><br /> 
                  <h2><span style="color:lightblue">questions?</span></h2><br />
                </section>

            </div>
        </div>

    <script type="text/javascript" src="lib/js/head.min.js"></script>
    <script type="text/javascript" src="js/reveal.js"></script>
		<script type="text/javascript" src="js/d3.min.js"></script>
		<script type="text/javascript" src="js/barchart.js"></script>

        <script>
            // Full list of configuration options available at:
            // https://github.com/hakimel/reveal.js#configuration
            Reveal.initialize({
                controls: true,
                progress: true,
                history: true,
                center: true,
                keyboard: true,
                transition: 'none', // none/fade/slide/convex/concave/zoom
                // Optional reveal.js plugins
                dependencies: [
                    { src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
                    { src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
                    { src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
                    { src: 'plugin/highlight/highlight.js', async: true, condition: function() { return !!document.querySelector( 'pre code' ); }, callback: function() { hljs.initHighlightingOnLoad(); } },
                    { src: 'plugin/zoom-js/zoom.js', async: true },
                    { src: 'plugin/notes/notes.js', async: true }
                ]
            });
        </script>

    </body>
</html>