<!doctype html>
<html lang="en">

    <!-- meta data and javascript import -->
	<head>
		<meta charset="utf-8">
		<title>Insights from a genetic isolate</title>
		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/base.css">
		<link rel="stylesheet" href="css/theme/black.css" id="theme">
		<!-- Code syntax highlighting -->
		<link rel="stylesheet" href="lib/css/zenburn.css">
		<!-- Make sure jquery is loaded before highcharts -->
		<!-- Jquery not required for Highstock charts which has lightweight jquery embeded -->
    <script type="text/javascript" src="js/jquery-1.11.3.min.js"></script>
    <script type="text/javascript" src="js/highcharts.js"></script>
		<!-- Import data.js to allow importing of csv files -->
		<!-- had issues with cross origin requests when accessing data stored locally -->
		<!-- Not currently implemmented in this presentation -->
		<script type="text/javascript" src="modules/data.js"></script>
		<!-- Creates dark themed plots throughout the presentation -->
		<script src="js/themes/gray.js"></script>
	</head>

    <!-- body content -->
    <body>

		<div class="reveal">

			<header class='headings'>
				<span></span>
			</header>

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
                <!-- title slide -->
  				      <section>
					          <h3>Insights from a Genetic Isolate</h3>
					          <p style="font-size: 20px">
						        <!-- <small> -->
						        <a href="http://sirselim.github.io/" target="blank">Miles Benton</a>
                    <!-- </small> -->
                    </p>
                    <p style="font-size: 14px">
                    Institue of Health and Biomedical Innovation, Queensland University of Technology, Brisbane, Australia <br>
                    </p>
                    <p style="font-size: 20px">
                      <a href="http://www.agtaconference.org/" target="blank">AGTA 2016 - Pullman Hotel, Auckland, New Zealand</a>
					          </p>
                    <p style="font-size: 16px">
                      9-12<sup style="font-size: 10px">th</sup> October 2016
					          </p>
                </section>

                <!-- presentation overview -->
                <section>
                  <h3>Overview</h3>

                  <ol>
                    <li>Norfolk Island</li><br />
                    <li><strike>3</strike> 2 'Case studies':</li>
                      <ul style="font-size: 24px;">
                        <p>
                        <li><strike>Phenomescan</strike></li>
                        <li>WGS, founder effects and glaucoma</li>
                        <li>Allele-specific methylation</li>
                        </p>
                      </ul>
                  </ol>
                </section>

                <!-- Genetic Isolates  -->
                <!--- Vertical slides take affect here following double section breaks -->
<!--                 <section>
                  <section>
                    <h2>Genetic Isolates</h2>
                  </section>
                  <section>
                    Here is more on isolates
                  </section>
                  <section>
                    Extended familys/pedigrees - identifying rare variants
                  </section>
                </section> -->

                <!-- Norfolk Island  -->
                <!--- Vertical slides take affect here following double section breaks -->
                <section>
                  <section>
                    <h2>Norfolk Island</h2>
                    <br />
                    ...and the Norfolk Island Health Study...<br>
                  </section>
                  <section>
                    <img src="images/NI_map.png" width=60% />
                  </section>
                  <section>
                    <img src="images/NorfolkIsland.png">
                  </section>
                  <section>
                    <img src="images/NIpedigree.png">
                    <ul>
                      <li> ~6000 member pedigree </li>
                    </ul>
                  </section>
                  <section>
                    <h3>Norfolk Island Health Study</h3>
                  </section>
                  <section>
                    <h3>Increased risk of metabolic disorders</h3>
                  </section>
                </section>

                <!-- Case 1: Phenomescan  -->
                <!--- Vertical slides take affect here following double section breaks -->
<!--                 <section>
                  <section>
                    <h2>1. Phenomescan</h2>
                    <br />
                    <p style="font-size: 20px">
                    Other contributors: <span style="color:lightblue">Rod Lea, Donia Macartney-Coxson, Michelle Hanna, David Eccles, Melanie Carless, Geoff Chambers, Claire Bellis, Harald Goring, Joanne Curran, Gregory Gibson, John Blangero, Lyn Griffiths</span>
                    </p>
                  </section>
                  <section>
                  <ul>
                    <li>Illumina 610-Quad SNP data for 500 core-pedigree members</li>
                    <li>initial GWAS of CVD-risk traits</li>
                    <li>applied a data reduction step (PCA)</li>
                    <li>derived 13 components</li>
                      <ul>
                        <li>9 of these with significant h<sup style="font-size: 28px">2</sup></li>
                      </ul>
                  </ul>
                  </section>
                  <section>
                    <center>
                      <img id="Fig1" src="images/Fig_1.png" width=80% />
                    </center>
                    <center>
                      performed GWAS on these 9 components
                    </center>
                  </section>
                  <section>
                    <center>
                      <img id="Fig2" src="images/Fig_2.png" hspace="0" width=48% />
                    </center>
                      <p style='text-align: left; font-size: 12px;'>Benton MC, Lea RA, Macartney-Coxson D, Bellis C, Carless MA, Curran JE, Hanna M, Eccles D, Chambers GK, Blangero J, Griffiths LR: <span style="color:lightblue"><i>Serum bilirubin concentration is modified by UGT1A1 Haplotypes and influences risk of Type-2 diabetes in the Norfolk Island genetic isolate.</i></span> <b>BMC Genetics</b>, 2015, 16:136.</p>
                  </section>
                  <section>
                    <center>
                      <img id="Fig3" src="images/Fig_3.png" width=75% />
                    </center>
                  </section>
                  <section>
                    <center>
                      <img id="Fig4" src="images/Fig_4.png" width=60% />
                    </center>
                    <p style='text-align: left; font-size: 12px;'>Benton MC, Lea RA, Macartney-Coxson D, Hanna M, Eccles DA, Carless MA, Chambers GK, Bellis C, Goring HH, Curran JE, Harper JL, Gibson G, Blangero J, Griffiths LR: <span style="color:lightblue"><i>A Phenomic Scan of the Norfolk Island Genetic Isolate Identifies a Major Pleiotropic Effect Locus Associated with Metabolic and Renal Disorder Markers.</i></span> <b>PLoS Genetics</b>, 2015, 11:e1005593.</p>
                  </section>
                  <section>
                    c
                  </section>
                </section> -->

                <!-- Case 2: WGS, founder effects and glaucoma  -->
                <!--- Vertical slides take affect here following double section breaks -->
                <section>
                  <section>
                    <h2>WGS, founder effects and glaucoma</h2>
                    <br />
                    <p style="font-size: 20px">
                    Other contributors: <span style="color:lightblue">Rod Lea, David Eccles, Donia Macartney-Coxson, Heidi Sutherland, Larisa Haupt, Lyn Griffiths</span>
                    </p>
                    <center>
                      <img id="rr" src="images/Rod.jpg">
                      <img id="db" src="images/Donia.jpg">
                      <img id="dv" src="images/David.png">
                      <!-- <img id="db" src="images/Lyn.jpg"> -->
                    </center>
                  </section>
                  <section>
                    <h3>Whole Genome Sequencing</h3>
                      <img id="hiseqX10" src="images/hiseq-x-ten.jpg" width=60% />
                      <p style='text-align: center; font-size: xx-small;'>&copy; 2016 Illumina, Inc.</p>
                      <p style="text-align: center; font-size: largest;">108 core-pedigree members</p>
                      <p style="text-align: center; font-size: largest;">Illumina HiSeq X10 at the Garvan Institute</p>
                      <p style="text-align: center; font-size: largest;">Decent amount of data (David Eccles)</p>
                      <p style="text-align: center; font-size: largest;">QUT HPC facility</p>
                  </section>
                  <section>
                  <h4>Total variants:</h4>
                    <img src="images/WGS_variants.png" width=35% /">
                  <h4>Functional* variants:</h4> 
                    <img src="images/WGS_funcvariants.png" width=35% />
                    <p style="text-align: center; font-size: medium;">*Functional = predicted to be damaging in 5 in-silico tests (SIFT, POLYPHEN2, MUTATIONTASTER, PROVEAN, MUTATION ASSESSOR)</p>
                  </section>
                  <section>
                    <h3>Founder Effect Variants</h3>
                    <ul>
                    <li> <i>ACOT4</i> (Acyl-CoA Thioesterase 4) </li>
                    <li> Regulation of lipid metabolism </li>
                    <li> Amino acid = p.TYR190ASN </li>
                    <li> Freq A = 0.0099 (EUR), 0 (AFR) and 0 (EAS) </li>
                    <li> Freq A in NI = 0.26 </li>
                    <li> Rare allele most likely came from England via Bounty Mutineers and then increased in frequency in NI due to founder effect </li>
                    </ul>
                  </section>
                  <section>
                    <h3>Founder Effect Variants</h3>
                    <img src="images/WGS_boxplots.png">
                  </section>
                  <section>
                    <h3>Glaucoma</h3>
                    <p>There is a higher than usual prevalence of Glaucoma on NI, we explored the WGS data for assocated variants</p><br />
                    <h5>Variant Filtering Strategy</h5>
                    <ol style="font-size: 30px;">                      
                      <li> filtered vcf files at 14% MAF </li>
                      <li> filtered based on genotypic models, both dominant (DOM) and allelic (ALL) </li>
                      <li> this resulted in 1083 for the DOM model and 1437 for the ALL model </li>
                      <li> the final filter was aimed to assess functional impact, we chose to go with SIFT and PROVEAN </li>
                      <li> this returned <b>1</b> variant </li>
                      </ol>
                  </section>
                  <section>
                    <h3>Variant information</h3>
                    <ul style="font-size: 30px;">
                      <li> reference allele = C </li>
                      <li> alternate allele = T </li>
                      <li> MAF CASES = 0.43 vs MAF CONTROLS = 0.05 </li>
                      <li> ALL p-value: CC_ALL=2.849<sup>-07</sup> </li>
                    </ul> 
                    <p>Predicted as deleterious/damaging</p>
                    <p>Gene has been previously associated with Retinoblastoma and Wilson's disease (copper deposits around the cornea)</p>
                  </section>
                </section>

                <!-- Case 3: Allelic-specific methylation  -->
                <!--- Vertical slides take affect here following double section breaks -->
                <section>
                  <section>
                    <h2>Allelic-specific methylation</h2>
                    (very recent data)<br /><br />
                    <p style="font-size: 20px">
                    Other contributors: <span style="color:lightblue">Rod Lea, Donia Macartney-Coxson, Nicole White, Daniel Kennedy, Heidi Sutherland, Larisa Haupt, Kerrie Mengersen, Lyn Griffiths</span>
                    </p>
                    <center>
                      <img id="rr" src="images/Rod.jpg">
                      <img id="db" src="images/Donia.jpg">
                      <img id="dv" src="images/Nicole.jpg">
                    </center>
                  </section>
                  <section>
                    <h5>Identification of allele-specific methylation profiles across generations</h5>
                    <span style="color:lightblue">Proof of principle pilot study</span><br />
                    <p>
                    <ul>
                      <li>measuring genome-wide allele-specific methylation (ASM)</li>
                        <ul>
                          <span style="font-size: 75%">
                          <li>NGS bisulphite sequencing</li> 
                          <li>SeqCap Epi CpGiant (Illumina HiSeq)</li>
                          </span>
                        </ul>
                        <br />
                      <li>collected data for <span style="color:lightblue"><b>24</b></span> NI individuals</li> 
                        <ul>
                          <span style="font-size: 75%">
                          <li>comprising a close 3 generation pedigree</li>
                          </span>
                        </ul>
                        <br />
                      <li>currently generating data for another <span style="color:lightblue"><b>~90</b></span> samples</li>
                    </ul>
                    </p>
                  </section>
                  <section>

                    <ul>
                      <li>Fully customised QC and analysis pipeline:</li>
                        <ul>
                          <li>fastqc, trimgalore</li>
                          <li>bismark, sambamba, picard tools</li> 
                          <li>PileOMeth, R and methylkit</li>
                        </ul>
                        <br />
                      <li>parallel processing enabled for local and remote machines</li>
                        <br />
                      <li>filtered at minimum 10 counts</li> 
                    </ul>
                  </section>
                  <section>
                    Initial metrics<br>
                    coverage<br>
                    <p style="font-size: 20px";>
                     - the lowest number of CpGs called for a given sample is 2.67M<br>
                     - the highest number of CpGs called for a given sample is 7.52M<br>
                     - the average across all samples is 3.48M<br>
                     - on-target mapping rate across the 24 samples >95% average<br>
                     - ~1.12M CpG sites in common across the 24 samples with at least 10 times coverage and on-target<br>
                    </p>
                  </section>
                  <section>
                    <h4>QC</h4>
                    <center>
                      <img id="genderPCA" src="images/NI_gender_PCA.png" width=85% />
                    </center>
                  </section>
                  <section>
                    <h4>QC</h4>
                    <center>
                      <img id="sampleQC" src="images/NI6828_SNP_compare_160816.png" width=95% />
                    </center> 
                    Good concordance between Illumina arrays and BS-seq                
                  </section>
                  <section>
                    <h2>Allelic-specific methylation</h2>
                    <ul>
                    <li>for an initial look-see used methpipe suite<br></li>
                    <p>
                      <ul>
                      <li>estimates ASM at each CpG site based on strand ratios of methylation</li>
                      </ul>
                    </p>
                    <li>moving forward we want to use read data:</li>
                    <p>
                      <ul>
                        <li>assign parent of origin and then infer ASM</li>
                        <li>gives the actual info rather than just knowning where an ASM event is</li>
                      </ul>
                    </p>
                    <li>promising method recently published for mice ASM - <span style="color:lightblue"><a href="https://github.com/FelixKrueger/SNPsplit" target="blank">SNPsplit (Krueger 2016)</a></span></li>
                  </ul>
                  </section>
                  <section>
                    <h3>ASM regions</h3>
                    <ul>
                      <li>Using a custom clustering method we identified ~1800 ASM regions (AMRs) conserved within the pedigree</li>
                      <li>Many of these AMRs map to known and predicted imprinted genes</li>
                    </ul>
                  </section>
                  <section>
                    <h2>Visualising ASM</h2>
                    <center>
                      <img id="chr7_ASM" src="images/example_asm_chr7.png" width=85% />
                    </center>
                      <p style="text-align: center; font-size: 18px;">
                      Documented and predicted human imprinted loci are displayed in <span style="color:darkred"><b>darkred</b></span>.
                    </p>
                  </section>
                  <section>
                    <center>
                      <img id="movie" src="images/scatter_movie.gif" width=60% />
                    </center>
                    <p style="text-align: center; font-size: 18px;">
                      ASM plot of chromosome 20 for a nuclear family (<span style="color:darkred"><b>father</b></span>, <span style="color:pink"><b>mother</b></span>, <span style="color:lightblue"><b>son</b></span>, <span style="color:green"><b>daugther</b></span>)
                    </p>
                  </section>
                  <section>
                  <h3>Presence of SNPs</h3>
                  <ul>
                  <li><span style="color:lightblue"><b>1,127,867</b></span> total methylation sites</li>                    
                  <li><span style="color:lightblue"><b>444,330</b></span> have a SNP present on C/G (<span style="color:lightblue"><b>39.4%</b></span>)</li>
                  </ul>
                  <p>
                  ...however...
                  </p>
                  <p>
                   Considering MAF:
                  <ul>
                    <li><span style="color:lightblue"><b>231,452</b></span> SNPs have recorded MAF info</li>
                    <li><span style="color:lightblue"><b>12,670</b></span> SNPs >= 0.05 MAF</li>
                    <li><span style="color:lightblue"><b>26,935</b></span> SNPs >= 0.01 MAF</li>
                  </ul> 
                  </p>
                  </section>
                  <section>
                    <center>
                      <img id="chr15_SNP" src="images/ASM_chr15_SNP_MAF.png" width=85% />
                    </center>
                  </section>
                  <section>
                    <center>
                      <img id="chr20_SNP" src="images/ASM_chr20_SNP_MAF.png" width=85% />
                    </center>
                  </section>
                  <section>
                    <center>
                      <img id="chr21_SNP" src="images/ASM_chr21_SNP_MAF.png" width=85% />
                    </center>
                  </section>
                  <section>
                    <center>
                      <img id="ploschr21" src="images/plos_chr21.png" width=85% />
                    </center>
                  </section>
                  <section>
                    <h3>Epigenetic inheritance</h3>
                    Next thing is to move onto modelling the inheritance...<br />
                    ...with the help of our brilliant Bayesian statisticians!<br>
                  </section>
                </section>

                <!-- Acknowledgements  -->
                <section>
                  <h3>Acknowledgements</h3>
                  <style type="text/css">
                  .tg  {border-collapse:collapse;border-spacing:0;border:none;margin:0px auto;}
                  .tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 20px;border-style:solid;border-width:0px;overflow:hidden;word-break:normal;}
                  .tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 20px;border-style:solid;border-width:0px;overflow:hidden;word-break:normal;}
                  .tg .tg-e3zv{font-weight:bold}
                  </style>
                  <table class="tg" style="undefined;table-layout: fixed; width: 780px">
                  <colgroup>
                  <col style="width: 260px">
                  <col style="width: 260px">
                  <col style="width: 260px">
                  </colgroup>
                    <tr>
                      <th class="tg-e3zv">QUT</th>
                      <th class="tg-e3zv">STDOI @ UTRGV<br></th>
                      <th class="tg-e3zv">NZ collaborators<br></th>
                    </tr>
                    <tr>
                      <td class="tg-031e">Lyn Griffiths<br></td>
                      <td class="tg-031e">John Blangero<br></td>
                      <td class="tg-031e">Donia Macartney-Coxson (ESR)<br></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">Rod Lea<br></td>
                      <td class="tg-031e">Joanne Curran<br></td>
                      <td class="tg-031e">David Eccles (Gringene Bioinformatics)<br></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">Larisa Haupt<br></td>
                      <td class="tg-031e">Harald Goring<br></td>
                      <td class="tg-031e">Geoff Chambers (VUW)</td>
                    </tr>
                    <tr>
                      <td class="tg-031e">Heidi Sutherland<br></td>
                      <td class="tg-031e"><br></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">Michelle Hanna<br></td>
                      <td class="tg-031e"><br></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">...the rest of the GRC IHBI lab group</td>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">Nicole White</td>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">Daniel Kennedy</td>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e">Kerrie Mengersen</td>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"></td>
                    </tr>
                  </table>
                  <br />
                  <table class="tg" style="undefined;table-layout: fixed; width: 780px">
                  <colgroup>
                  <col style="width: 260px">
                  <col style="width: 260px">
                  <col style="width: 260px">
                  </colgroup>
                    <tr>
                      <td class="tg-031e">Garvan Institute of Medical Research / Kinghorn Centre for Clinical Genomics</td>
                      <td class="tg-031e">Melanie Carless (Texas Biomedical Research Institute)</td>
                      <td class="tg-031e">NHMRC</td>
                    </tr>
                    <tr>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"><br>Claire Bellis (Genome Institute of Singapore)</td>
                      <td class="tg-031e"></td>
                    </tr>
                    <tr>
                      <td class="tg-031e"></td>
                      <td class="tg-031e"><br>Greg Gibson (Georgia Institute of Technology, USA)</td>
                      <td class="tg-031e"></td>
                    </tr>
                  </table><br />
                  <span style="color:lightblue; font-size:20px"><b>The people of Norfolk Island who who volunteered for this study.</b></span>
                </section>

                <!-- last slide, questions -->
                <section>
                  <h2>Thank you</h2><br /> 
                  <h2><span style="color:lightblue">questions?</span></h2>
                </section>

            </div>
        </div>

    <script type="text/javascript" src="lib/js/head.min.js"></script>
    <script type="text/javascript" src="js/reveal.js"></script>
		<script type="text/javascript" src="js/d3.min.js"></script>
		<script type="text/javascript" src="js/barchart.js"></script>

        <script>
            // Full list of configuration options available at:
            // https://github.com/hakimel/reveal.js#configuration
            Reveal.initialize({
                controls: true,
                progress: true,
                history: true,
                center: true,
                keyboard: true,
                transition: 'slide', // none/fade/slide/convex/concave/zoom
                // Optional reveal.js plugins
                dependencies: [
                    { src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
                    { src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
                    { src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
                    { src: 'plugin/highlight/highlight.js', async: true, condition: function() { return !!document.querySelector( 'pre code' ); }, callback: function() { hljs.initHighlightingOnLoad(); } },
                    { src: 'plugin/zoom-js/zoom.js', async: true },
                    { src: 'plugin/notes/notes.js', async: true }
                ]
            });
        </script>

    </body>
</html>