<!DOCTYPE html>
<html>
  <head>
    <title>Genome informatics for translation to clinical diagnostics</title>
    <meta charset="utf-8">
    <meta name="author" content="Dr Miles Benton" />
    <script src="libs/htmlwidgets/htmlwidgets.js"></script>
    <script src="libs/viz/viz.js"></script>
    <link href="libs/DiagrammeR-styles/styles.css" rel="stylesheet" />
    <script src="libs/grViz-binding/grViz.js"></script>
    <link rel="stylesheet" href="libs/remark-css/example_mod.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Genome informatics for translation to clinical diagnostics
## <h3 id="user-friendly-web-based-tools-bring-sequence-data-to-the-clinic-nhmrc-new-investigator-grant---pitch">“<em>user-friendly web-based tools bring sequence data to the clinic</em>”<br/><br/>NHMRC New Investigator Grant - Pitch</h3>
### <a href="http://sirselim.github.io/">Dr Miles Benton</a>
### 2017/07/28

---




# .center[Motivation]

.center[&lt;span&gt;**NGS technology shows great promise for understanding genetic basis of disease and personalised medicine**&lt;/span&gt;]
.center[.......but....... ]

--

**identification of known or novel pathogenic variants**

  - found among a host of common and rare polymorphisms
  - identification of clinically relevant variants is time consuming 
    + large potential for analysis induced false negatives
  - larger datasets (WES/WGS) contain [**thousands**]() to [**millions**]() of variants
  - aggravated by complex conditions/disorders 
    + multitudes of genes / mutations responsible for symptoms or important for treatment

--

**current tools are limited and specialised**

--

**making data available (interpretable) to end user ie. clinician/patient**

--

**visualisation and interactivity incredibly powerful tools ('big data')**

--

.center[**I'm an emerging researcher (3.5 years out from PhD)**]

---
class: inverse
# .center[Major drivers for change] 

**paywalls**

  - nearly all software suites are licensed == '$$$'
  - might not be an issue for institutes, harder for non-research

--
&lt;br/&gt;
&lt;br/&gt;

**'blackbox' software**

  - mysterious to users
  - difficult/impossible to modified (closed-source)
    + those not closed source are tailored to specific lab/job (i.e. CPIPE)
  - can be hard to use / require specific skills
    + not everyone is a bioinformatician&lt;br/&gt;
  
--
&lt;br/&gt;
&lt;br/&gt;

.large[.center[**NO software readily accessible and user-friendly for end users 'on the ground'**]]

---
class: top
# .center[Significance]

.center[current methods fall into either:]

.pull-left[
  .center[**pay to use**]
]

.pull-right[
  .center[**free but tricky**]
]

--
&lt;br/&gt;

.center[**open-source software is the backbone of bioinformatics**]

--

.center[Accessible, flexible, powerful, and ... free]

<div id="htmlwidget-b9248d498223fde31a0f" style="width:750px;height:200px;" class="grViz html-widget"></div>
<script type="application/json" data-for="htmlwidget-b9248d498223fde31a0f">{"x":{"diagram":"\ndigraph a_nice_graph {\n\n# node definitions with substituted label text\nnode [fontname = Helvetica]\na [label = \"open source\"]\nb [label = \"community\"]\nc [label = \"better tools\"]\nd [label = \"better functionality\"]\n\n# edge definitions with the node IDs\na -> {b}\nb -> {a}\nb -> {c d}\n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>

--
&lt;br/&gt;

.center[**wide benefit**: research | diagnostic | clinical | public ]

---

# .center[Aims]

**address shortfall in current methods**
  + free and simple

--
  
**[Shiny](https://shiny.rstudio.com/) integration**
  + the power of [R](https://www.r-project.org/)/[RStudio](https://www.rstudio.com/)
  + visualisation/interactivity front and center

--
  
**ease of use**
  + want it to work in the hands of all end users
    - clinic
    - research
    - public?

--
  
**modularity** (full extensible and upgradable)

--
  
**cloud integration and deployment options**

---
class: top inverse
# .center[VCF files are powerful but clumsy&lt;sup&gt;\*&lt;/sup&gt;]
.center[\[\*\] *if you are not familiar with the commandline*]

To 95% of people these are just really large complex text files

  - no easy interaction
  - no ready visualisation

--

&lt;br/&gt;
  
# .center[variant filtering process is complicated&lt;sup&gt;\*\*&lt;/sup&gt;]
.center[\[\*\*\] *even if you **are** familiar with the commandline*]

.center[A large number of people still do this manually...]
.center[**...this is what computers are for!**]

---
background-image: url(images/pipeline_chart.png)
background-position: 50% 75%
background-size: 75%
class: top
# .center[Progress to date]

.center[[WESTARC]() - live demonstration]

---
class: top inverse
# .center[Research Design]

**Develop a series of modules, each achieving a specific task:  **

  - initial QC and sequence alignment  
    &lt;span style="font-size: 75%"&gt;+ including functionality for structural variation (routinely overlooked)&lt;/span&gt;

  - VCF annotation and manipulation  
    &lt;span style="font-size: 75%"&gt;+ currently only accessible to 'advanced' users&lt;/span&gt;
  
  - simple interactive 'base' fontend (i.e. WESTARC)  
    &lt;span style="font-size: 75%"&gt;+ include database interfacing&lt;/span&gt;
  
  - Additional analysis interface  
    &lt;span style="font-size: 75%"&gt;+ phenotype, case/control, sample comparison ...&lt;/span&gt;

--

**Provide to clinical end users:  **
  - have access to several engaged clinical geneticists (testing group?)

--

**Distribute:  **
  - GitHub, docker, &amp; online cloud server (Amazon S3)

---
class: top
# .center[Expected outcomes]

&gt; *simple, scalable and robust method for the annotating and categorising genetic variants enabling more rapid and effective analysis of potentially pathogenic variants*

--

**A fully operational suite of software 'modules': **
  - integrate into an easy-to-use workflow
  - can handle all forms of sequence data
  - open-source / free to use and develop 

**Deployment of an user friendly app version / suite of apps:  **
  - integration with existing databases
  - cloud deployment
  - docker integration

**Direct to consumer:  **
  - putting the **'power'** back in the hands of those that matter

---
class: middle inverse
# .center[Team members on this submission]

.center[**Dr Miles Benton** - Principal Investigator (emerging investigator)&lt;sup&gt;[1]()&lt;/sup&gt;]

.pull-left[
  
.right[
Prof Lyn Griffiths&lt;sup&gt;[1]()&lt;/sup&gt;  
A/Prof Rod Lea&lt;sup&gt;[1]()&lt;/sup&gt;  
Dr Robert Smith&lt;sup&gt;[1]()&lt;/sup&gt;  
Prof Greg Gibson&lt;sup&gt;[2]()&lt;/sup&gt;
]
]

.pull-right[
  
AI (*Human Genetics*)&lt;sup&gt; &lt;/sup&gt;  
AI (*Genome Informatics*)&lt;sup&gt; &lt;/sup&gt;  
AI (*Diagnostics*)&lt;sup&gt; &lt;/sup&gt;  
AI/Mentor (*Integrative Genomics*)&lt;sup&gt; &lt;/sup&gt;

]

.footnote[
[[1]()] CDA, MM, IHBI, QUT  
[[2]()] Georgia Tech, USA
]
    </textarea>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "4:3"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {window.dispatchEvent(new Event('resize'));});</script>

<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  tex2jax: {
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre']
  }
});
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
