<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
  <head>
    <title>.huge[VCF-DART]</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="libs/remark-css/example_mod.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# .huge[VCF-DART]
## A Customisable Analysis Pipeline for the Identification of Clinically Relevant Genetic Variants in Next Generation Sequencing Data
### <div class="line-block"><a href="http://sirselim.github.io/">Dr Miles Benton</a><br />
Senior Scientist Bioinformatics<br />
Institute of Environmental Science and Research (ESR)<br />
<br />
.large[HGSA Wellington, 3<small><sup>rd</sup></small>-6<small><sup>th</sup></small> August 2019]<br />
<br />
<br />
.center[<img src="images/ESR_SFC_logo.png" style="width: 25%; border: 3px solid white;"/>]</div>

---

class: middle



# .center[Advocate for reproducible research...]

## .center[Where possible my presentations and code are available online]

&lt;br /&gt;

&lt;p&gt;
.center[
&lt;img src="images/github_logo.png" style="width: 520px; margin-right: 1%; margin-top: 1.5em;"/&gt;
&lt;img src="images/sirselim_qrcode.png" style="width: 202px; margin-right: 1%; margin-top: 1.5em;"/&gt;
]
&lt;/p&gt;

&lt;br /&gt;

.center[[sirselim.github.io/presentations](http://sirselim.github.io/presentations)]
  
---
class: middle inverse

# QUT (Feburary 2014 - June 2018)

## Brisbane (QLD), Australia

### &lt;span style="color:#3498DB"&gt;PostDoc Research Fellow&lt;/span&gt;

&lt;p&gt;
.center[
&lt;img src="images/qut_ihbi.jpg" style="width: 240px; margin-right: 1%; margin-top: 1.5em; border: 3px solid white;"/&gt;
&lt;img src="images/qut_gardens.jpg" style="width: 240px; margin-right: 1%; margin-top: 1.5em; border: 3px solid white;"/&gt;
]
&lt;/p&gt;

---
layout: false

background-image: url("images/proton2.jpg")
background-size: cover

---
class: middle

# Diagnostics Lab

.large[Gene panel and Exome Sequencing (NATA accredited - neurological disorders)]

&lt;br&gt;

## Sanger -&gt; Gene panel -&gt; Exome

* used to manually work through 30-60K rows of data in Excel!
* could take 4-8 weeks to identify and validate variant(s)

---
class: middle inverse

# VCF-DART - what it's not...

It is **NOT** a variant calling and quality control pipeline

... it's about data

---

# Overview

&lt;p&gt; 
.center[&lt;img src="images/Pipeline_flowchart_v2.png" style="width: 95%; margin-right: 1%; margin-top: -0.5em; border: 3px solid white;"/&gt;]
&lt;/p&gt;

---

# Variant tiers

&lt;p&gt; 
.center[&lt;img src="images/tiers.png" style="width: 70%; margin-right: 1%; margin-top: -0.5em; border: 3px solid white;"/&gt;]
&lt;/p&gt;

---
class: middle inverse

# "Module 1" - VCF-DART

.large[(&lt;span style="color:#3498DB"&gt;**V**&lt;/span&gt;ariant &lt;span style="color:#3498DB"&gt;**C**&lt;/span&gt;all &lt;span style="color:#3498DB"&gt;**F**&lt;/span&gt;ormat - &lt;span style="color:#3498DB"&gt;**D**&lt;/span&gt;iagnostic &lt;span style="color:#3498DB"&gt;**a**&lt;/span&gt;nd &lt;span style="color:#3498DB"&gt;**R**&lt;/span&gt;eporting &lt;span style="color:#3498DB"&gt;**T**&lt;/span&gt;ool)]

---
layout: false

background-image: url("images/vcfdart_1.png")
background-size: contain

---
layout: false

background-image: url("images/vcfdart_2.png")
background-size: contain

---
class: middle

# Runtime

.huge[30-50K variants (avg Proton exome VCF):]

  - 6-10 mins (24 core 256GB RAM)
  - 17-23 mins (4 core 12GB RAM)

---
class: middle inverse

# "Module 2" - VCF-DART Viewer

---
class: middle center

.huge[[...live demonstration...](https://43.240.99.48/sample-apps/WES_ShinyDiscover/)]

&lt;br&gt;

.huge[**?**]

---
layout: false

background-image: url("images/backup_1.png")
background-size: contain

---
layout: false

background-image: url("images/backup_2.png")
background-size: contain

---
layout: false

background-image: url("images/backup_3.png")
background-size: contain

---
layout: false

background-image: url("images/backup_4.png")
background-size: contain

---
layout: false

background-image: url("images/backup_5.png")
background-size: contain

---
layout: false

background-image: url("images/backup_6.png")
background-size: contain

---
class: 

&lt;p&gt; 
.center[&lt;img src="images/nectarcloud.png" style="width: 70%; margin-right: 1%; margin-top: -0.5em; border: 3px solid white;"/&gt;]
&lt;/p&gt;

Spun up VM to give reviewers a chance to ~~break~~ test things.

* using 6 public exomes (1000G)

&lt;br&gt;

--

.large[**Allocated VM:**]

* Ubuntu 18.08
* 4 cores
* 12GB RAM
* 30GB root
* 120GB ephemeral disk

--

&lt;br&gt;

...so it can scale down to quite reasonable specs...

---
class: middle

## VCF-DART in the lab

* now can have a short list of variants in &lt;span style="color:#3498DB"&gt;**1 day**&lt;/span&gt; 
* validation in &lt;span style="color:#3498DB"&gt;**&lt;2 weeks**&lt;/span&gt;

## Summary

* open source
* modular (tools and databases)
* scalable (laptops -&gt; servers)
* easy to deploy and use

---
class: middle inverse

## Current state of play

* exciting times!

* Auckland University (Prof Cristin Print), Genomics Aotearoa

* merging 'pipelines' to offer a flexible, unified and freely distributable system across NZ

* accreditation

&lt;p&gt;
.center[
&lt;img src="images/UoA.png" style="width: 272px; margin-right: 1%; margin-top: 1.5em; border: #FFFFFF  3.5px outset;"/&gt;
&lt;img src="images/GA-Wide-Colour-1200px.jpg" style="width: 315px; margin-right: 1%; margin-top: 1.5em; border: #FFFFFF  3.5px outset;"/&gt;
]
&lt;/p&gt;

---
class: middle

## Availability

&lt;br&gt;

.large[Working on a wide range of deployment options:]

&lt;br&gt;

&lt;p&gt;
.center[
&lt;img src="images/github.jpg" style="width: 200px; margin-right: 2%;"/&gt;
&lt;img src="images/aws.png" style="width: 215px; margin-right: 2%;"/&gt;
&lt;img src="images/singularity-logo.svg" style="width: 135px; margin-right: 2%; margin-left: 2%;"/&gt;
&lt;img src="images/NESI-logo.jpg" style="width: 300px; margin-right: 2%;"/&gt;
]
&lt;/p&gt;

---
class: middle inverse

# to-do

* &amp;#10004; ~~submit~~ manuscript - accepted (The Journal of Molecular Diagnostics) 
* &amp;#10004; [release to the wild - GitHub (anyone interested can help out)](https://github.com/sirselim/WES_ShinyDiscover)
* finish implementing ClinGen links
* option for GnomAD-beta
* refactoring main code base (snakemake)
* complete docker/singularity version
* continue developing documentation!
* work with clinicians to further develop and refine

---
class: top

# acknowledgements

&lt;div id="images"&gt;
    &lt;a href="http://xyz.com/hello"&gt;
        &lt;img src="images/robert.jpg" width="180px" height="250px"&gt;
        &lt;div class="caption"&gt;Dr Robert Smith (QUT)&lt;/div&gt;
    &lt;/a&gt;
    &lt;a href="http://xyz.com/hi"&gt;
        &lt;img src="images/donia.jpg" width="180px" height="250px"&gt; 
        &lt;div class="caption"&gt;Dr Donia Macartney-Coxson (ESR)&lt;/div&gt;
    &lt;/a&gt;
&lt;/div&gt;


.pull-left[.medium[
Craig Windell (QUT)

QUT lab group

Sam Beardman 
]]

.pull-right[.medium[
Prof Cristin Print (UoA)

Dr Ben Curran (UoA)

QRISCloud / Nectar 
]]

---
class: middle inverse

&lt;p&gt;
.huge[.center[&lt;b&gt;Thank you!&lt;/b&gt;]]
&lt;br&gt;
&lt;/p&gt;
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();</script>

<script>
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
